Modules/resource.c

```diff
@@ -340,144 +340,136 @@ resource_methods[] = {
 /* Module initialization */
 
 
-static struct PyModuleDef resourcemodule = {
-    PyModuleDef_HEAD_INIT,
-    "resource",
-    NULL,
-    -1,
-    resource_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-PyMODINIT_FUNC
-PyInit_resource(void)
+static int resource_exec(PyObject *module)
 {
-    PyObject *m, *v;
+#define ADD_INT(module, value)                         \
+    do {                                               \
+        if (PyModule_AddIntMacro(module, value) < 0) { \
+            return -1;                                 \
+        }                                              \
+    } while (0)                                        \
```

Comment:
Remove the last trailing \:

```suggestion
    } while (0)
```

Reply:
done, thanks.